js/app.js

```diff
@@ -14,13 +14,27 @@ function TextApp() {
   this.windowController_ = null;
 
   this.hasFrame_ = false;
+
+  /**
+   * If true, the "default" setting for the color theme chooses light/dark
+   * depending on the system's preferred color scheme.
+   * If false, the "default" setting is a dark side nav and a light editor.
+   *
+   * @type {boolean}
+   */
+  this.enableSystemTheme_ = false;
+  /** The media query list to detect if the preferred color scheme is dark. */
+  this.colorSchemeMatcherDark_ = null;
 }
 
 /**
  * Called when all the resources have loaded. All initializations should be done
  * here.
  */
 TextApp.prototype.init = function() {
+  this.enableSystemTheme_ =
+      parseInt(navigator.appVersion.match(/Chrome\/(\d+)\./)[1], 10) >= 103;
```

Comment:
Looks good, maybe double check with patti who did something similar about if there are any pitfalls we need to consider here.

Reply:
The version check is copied from https://github.com/GoogleChromeLabs/text-app/blob/ae27c62307a60d6f469e1c8a1e5a454e89b08169/js/background.js#L15

I checked with Patti. She said there will be a reduction in the user agent eventually, but it won't be an issue because this code only extracts the major chrome version. https://blog.chromium.org/2021/05/update-on-user-agent-string-reduction.html

It works on the unified format: https://www.chromium.org/updates/ua-reduction/#unified-format

Reply:
beaut! thanks for checking :)